com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```diff
@@ -75,6 +76,60 @@ cleanup:
   - /lib/wine/*.def
   - /lib32/wine/*.def
 modules:
+  - name: protonfixes
+    buildsystem: simple
+    build-commands:
+      - |
+        cp -a \
+          protonfixes \
+        ${FLATPAK_DEST}
+    sources:
+      - *proton_source
+
+
+  - name: fonts
+    buildsystem: autotools
+    no-autogen: true
+    make-args:
+      - fonts
+    no-make-install: true
+    post-install:
+      - find obj-fonts/ -xtype f \( -name "*.otf" -o -name "*.ttf" \) -ls -exec install -Dm0644 -t ${FLATPAK_DEST}/share/fonts/ {} \;
+    sources:
+      - *proton_source
+      - type: shell
+        commands:
+          - ./configure.sh --no-steam-runtime
+    modules:
+      - name: fontforge
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DCMAKE_BUILD_TYPE=Release
+          - -DBUILD_SHARED_LIBS=OFF
+          - -DENABLE_GUI=OFF
+          - -DENABLE_NATIVE_SCRIPTING=ON
+          - -DENABLE_PYTHON_SCRIPTING=OFF
+          - -DENABLE_PYTHON_EXTENSION=OFF
+          - -DENABLE_LIBSPIRO=OFF
+          - -DENABLE_LIBUNINAMESLIST=OFF
+          - -DENABLE_LIBGIF=OFF
+          - -DENABLE_LIBJPEG=OFF
+          - -DENABLE_LIBPNG=OFF
+          - -DENABLE_LIBREADLINE=OFF
+          - -DENABLE_LIBTIFF=OFF
+          - -DENABLE_WOFF2=OFF
+          - -DENABLE_DOCS=OFF
+        cleanup:
+          - "*"
+        sources:
+          - type: archive
+            url: "https://github.com/fontforge/fontforge/releases/download/20201107/fontforge-20201107.tar.xz"
+            sha256: 68bcba8f602819eddc29cd356ee13fafbad7a80d19b652d354c6791343476c78
+      - files/fonts/afdko-dev.json
+      - files/fonts/afdko.json
```

Comment:
My approach was to name additional files by the upstream project name they're related to. I.e. these would be something like `files/afdko/python3-setuptools_scm.json` and `files/afdko/python3-afdko.json`. But that's just a personal preference.

Reply:
I've named them after `requirements.txt` and `requirements-dev.txt` in `afdfko` source tree.

Reply:
Maybe `files/afdko/python3-requirements-dev.json` and `files/afdko/python3-requirements.json`, then?

Reply:
I'm okay to rename the files, but I'm reluctant to redo a full build just for this. Is this okay if I cancel the test build and just merge this given that the previous build worked ?

Reply:
I guess so, if you don't plan to add any more changes.

Reply:
No more change, from my POV, it's all good.